src/plugins/vis_type_timeseries/public/application/components/color_rules.tsx

```diff
@@ -54,33 +62,45 @@ const defaultPrimaryName = i18n.translate('visTypeTimeseries.colorRules.defaultP
   defaultMessage: 'background',
 });
 
-const operatorOptions = [
+export const colorRulesOperatorsList: ColorRulesOperator[] = [
   {
     label: i18n.translate('visTypeTimeseries.colorRules.greaterThanLabel', {
       defaultMessage: '> greater than',
     }),
-    value: 'gt',
+    method: Operator.Gt,
   },
   {
     label: i18n.translate('visTypeTimeseries.colorRules.greaterThanOrEqualLabel', {
       defaultMessage: '>= greater than or equal',
     }),
-    value: 'gte',
+    method: Operator.Gte,
   },
   {
     label: i18n.translate('visTypeTimeseries.colorRules.lessThanLabel', {
       defaultMessage: '< less than',
     }),
-    value: 'lt',
+    method: Operator.Lt,
   },
   {
     label: i18n.translate('visTypeTimeseries.colorRules.lessThanOrEqualLabel', {
       defaultMessage: '<= less than or equal',
     }),
-    value: 'lte',
+    method: Operator.Lte,
+  },
+  {
+    label: i18n.translate('visTypeTimeseries.colorRules.emptyLabel', {
+      defaultMessage: 'empty',
+    }),
+    method: Operator.Empty,
+    isValueConstant: true,
```

Comment:
I would name this flag a little bit different. It has confused me a bit tbh.
You can name it something like `hideValueSelector` or `needValue: false` or something like that.

Reply:
Okey, I'll change it.